.github/workflows/build.yaml

```diff
@@ -1,8 +1,98 @@
 name: Build
 on:
+  schedule:
+    # Run nightly at 00:00
+    - cron: '0 0 * * *'
+
   workflow_dispatch:
+    inputs:
+      otp-ref-name:
+        required: true
+      otp-ref:
+        required: true
+      openssl-version:
+        default: "3.1.6"
+      wxwidgets-version:
+        default: "3.2.6"
 jobs:
-  build:
+  schedule:
+    name: "Schedule nightly"
+    if: github.event_name == 'schedule'
     runs-on: ubuntu-latest
```

Comment:
Would it make sense to version this? I've had cases where CI broke for similar reasons (using latest). Or maybe we want it and we'll deal with issues once they happen (?)

Reply:
Was just a detail, in any case, can be dealt with later. What'll probably be important is to have a "Job failed" issue, but I already created an issue to deal with that later.